jbpm-human-task/jbpm-human-task-jpa/pom.xml

```diff
@@ -75,6 +75,11 @@
     </dependency>
 
     <!-- TEST -->
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-test-util</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This was picked up due to a search and replace: https://github.com/kiegroup/jbpm/tree/master/jbpm-human-task/jbpm-human-task-jpa/src/test/resources

If you are sure it's not used I will remove the src/test/resources folder and the pom updates